config/custom-environment-variables.json

```diff
@@ -4,5 +4,14 @@
     },
     "settings": {
       "adminAccounts": "ADMIN_PASSWORD"
+    },
+    "api_key" : {
+        "SENDGRID": "SENDGRID_API_KEY"
+    },
+    "recipient" : {
```

Comment:
and this `custom-environment-variables.json` file is only to connect external configs (Heroku Configs, this case) to our nodejs app's config. (like a link/bridge between them)

So this information shouldn't be here and enough to be in `default.json`.